src/plugin/sdl/configure.ac

```diff
@@ -7,6 +7,20 @@ AM_PATH_SDL2($SDL_VERSION,,
 	    AC_MSG_ERROR([*** SDL version $SDL_VERSION not found!])
 )
 
+dnl Check for SDL_ttf
+AC_CHECK_LIB(SDL2_ttf, TTF_Init, [
+  SDL_LIBS+=" -lSDL2_ttf"
+], [
+  AC_MSG_ERROR([SDL TTF development package not installed])
```

Comment:
Would be better to make this a warning
and a soft-dep. But not a high prio task.
Can be adjusted any time later (after merge).

Reply:
Removed the hard requirement for TTF / fontconfig and used ifdefs for conditional compilation of TTF code.